nodes/kitchen-node/src/service.rs

```diff
@@ -71,7 +71,7 @@ macro_rules! new_full_start {
 }
 
 /// Builds a new service for a full client.
-pub fn new_full<C: Send + Default + 'static>(config: Configuration<C, GenesisConfig>)
+pub fn new_full(config: Configuration<GenesisConfig>)
```

Comment:
@jimmychu0807This is the key issue that I was stuck on. Can you say a bit about the change you've made? What does it do? Why is the way I had it wrong?

@cecton This is what I was asking for help with. Would appreciate your feedback / review on this change too.

Reply:
And thank you for the new CLI API. I think I will like it :)

Reply:
yw \o/

There used to be 3 type parameters: G for the RuntimeGenesis, E for ClientSpec's extension and C for a custom configuration to run the node.

G and E stay but C is gone. Your code still had C and it was passed in place of G (GenesisConfig) and GenesisConfig was used in E (which doesn't implement extension).

Custom configuration should now be passed by passing arguments to the builder function.